packages/bitcore-wallet-client/src/lib/credentials.ts

```diff
@@ -63,14 +64,15 @@ export class Credentials {
   use145forBCH: any;
   addressType: string;
   keyId: string;
-
+  token: string;
```

Comment:
`token?: string` will not always be available for every wallet.

Reply:
https://github.com/bitpay/bitcore/pull/2562